src/temporal/t.rast.import.netcdf/t.rast.import.netcdf.html

```diff
@@ -0,0 +1,110 @@
+<h2>DESCRIPTION</h2>
+
+<em>t.rast.import.netcdf</em> imports content of one or more NetCDF files into a
+GRASS GIS Space Time Raster Dataset (STRDS). NetCDF files are expected to follow
+the CF-convention. Files not adhering to those standards may fail to import.
```

Comment:
Add a link to the standards perhaps?

Reply:
Link to the cf-convention added in [387c75c](https://github.com/OSGeo/grass-addons/pull/794/commits/387c75c8ebbeb21c10e2052ef01081fb25818978)